core/config.js

```diff
@@ -0,0 +1,58 @@
+Nuxeo = Nuxeo || {};
+Nuxeo.UI = Nuxeo.UI || {};
+Nuxeo.UI.config = Nuxeo.UI.config || {};
+const { config } = Nuxeo.UI;
+
+Object.assign(config, {
+  /**
+   * Returns a property for a given `path`. If it is not defined, `fallback` is returned instead.
+   * If `fallback` is defined, the method will try to convert the property value to the same data type of `fallback`.
+   */
+  get(path, fallback) {
+    let val = path.split('.').reduce((a, b) => a && a[b], this);
+    if (fallback !== undefined && typeof val !== typeof fallback) {
```

Comment:
we don't really need the `fallback !== undefined` do we? you even handle it explicitly in the switch statement...

Reply:
I will remove it from the switch statement, and I thin the check should be `value !== undefined` instead.